### PR TITLE
Update contribution guidelines

### LINKED ISSUE #31: Add a contribution guide
Add some documentation explaining how users can contribute via the various possiblities:
- git fork / clone / PR workflow
- PR from the browser

### PR DESCRIPTION
**Pull request by gwarf**
Move contribution guidelines to repository root, include documentation about contributing via the browser.
Fix #31 .

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -19,22 +19,37 @@ including a wider community scope.
 
 ## Contribution Process
 
+All contributions have to go through a review process, and contributions can be
+made in two ways:
+
+- for simple contribution you can contribute from your browser by clickink the
```
Comment (enolfc) [inline CONTRIBUTING.md:25]:
```suggestion
- for simple contribution you can contribute from your browser by clicking the
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -64,7 +79,7 @@ You should submit your patch as a git branch named after the Github issue, such
 as `#3`\. This is called a _topic branch_ and allows users to associate a
 branch of code with the issue.
 
-It is a best practice to have your commit message have a _summary line_ that
+It is the best practice to have your commit message have a _summary line_ that
 includes the issue number, followed by an empty line and then a brief
```
Comment (enolfc) [inline CONTRIBUTING.md:83]: Should we enforce always having an issue first?

Comment (gwarf) [reply CONTRIBUTING.md:83]: I don't think it's that useful, in fact PR are already some kind of issues, and probably enough for now :)

Comment (gwarf) [reply CONTRIBUTING.md:83]: Even GitHub is joining them under https://github.com/EGI-Foundation/documentation/issues?q=is%3Aopen+

**Pull request merged**